services/connections.go

```diff
@@ -48,15 +48,22 @@ func (s *Linkedinserver) ConnectWithOtherUser(ctx context.Context, in *pb.Connec
 	Userslice = append(Userslice, user1)
 	Userslice = append(Userslice, user2)
 	conn, err := s.Db.ConnectWithOtherUserDbinteraction1(Userslice)
+	fmt.Println(conn)
 	if err != nil {
 		panic(err.Error())
 	}
+	var message string = "pending"
 	if !reflect.DeepEqual(conn, conn1) && conn.Status == "pending" {
+
 		s.Db.ConnectWithOtherUserDbinteraction2(Userslice)
-		Email.SendEmail(usermodel.Email)
+		// Email.SendEmail(usermodel.Email)
```

Comment:
why did you comment this out?

Reply:
its sending email each time i run so commented it out